src/components/SocialLinks-Color/index.js

```diff
@@ -30,7 +30,7 @@ const SocialLinksColor = () => {
           >
             <img height="30px" src={mail_icon} alt="mail" />
           </a>
-          <a href="http://slack.layer5.io/" target="_blank" rel="noreferrer">
+          <a href="https://slack.layer5.io/" target="_blank" rel="noreferrer">
```

Comment:
```suggestion
          <a href="https://slack.layer5.io/" target="_blank">
```

Let's make a point of leaving out  `rel="noreferrer"` on links within the layer5.io domain.